Userland/Libraries/LibGUI/MessageBox.cpp

```diff
@@ -81,7 +81,14 @@ void MessageBox::build()
 {
     auto& widget = set_main_widget<Widget>();
 
-    int text_width = widget.font().width(m_text);
+    int max_text_width = 0;
+    auto text_lines = m_text.split('\n');
+    auto number_of_lines = text_lines.size();
+    for (auto& text_line : text_lines) {
+        int text_width = widget.font().width(text_line);
```

Comment:
calculating proper width is done here: https://github.com/SerenityOS/serenity/pull/8490

Reply:
Nice work! I wonder if it makes more sense to calculate height in LibGfx as well.